### ISSUE TITLE
CLI doesn't handle spaces well

### ISSUE THREAD

**Issue by Parth** [lockbook/lockbook#2606]
Probably the way we're invoking vim is not right and ignoring quoted arguments or something

requested by @alexwasserman

**Comment by alexwasserman** [comment]
FYI - there's already #2210, but the app seems to handle spaces in file names ok, but the CLI doesn't

**Comment by Parth** [comment]
yeah I believe #2210 is from a time when the apps were discouraging spaces in their UX, but as we can't see those fields (encrypted) and people actually do want spaces (@CoreyCole & You?) in their names we're starting to move away from that incorrect assumption

**Comment by Parth** [comment]
`\ ` doesn't work either
